### ISSUE TITLE
General Page Hero Banner

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1363]
**Component block:** None, included in all templates

This story is likely covered through all the global styling changes, but will cover requirements so we can properly test the accessibility. 

**Design:** https://www.figma.com/design/T2bgro0OiuLqxvm1uLhJZ2/Clark-County-NV?node-id=2588-3761&t=PlFgmbRxMOkD329b-0

As a site visitor, I want the ability to access the content in the hero banner of a pages via visual display, keyboard, and/or screen reader and recognize which pages I am on and what content I will view

**Requirements:** 
Adjust hero sizing to begin just below the header - *shorten hero banner size, it should not extend lower on the page than current
Hero banner should meet color contrast WCAG AA requirements with: 
1. Hero banner should include dark overlay (35% opacity overlay)
Hero banner title should be only H1 on the page
Hero title should be styled according to brand guidelines (Title XXL style)
Hero banner should be readable by screen reader with all appropriate heading tags

**Acceptance Criteria:** 
2. Given I am accessing a web page, when the page loads, then the hero banner includes a dark overlay over the banner image. 
3. Given I am inspecting a web page, when the page includes a hero banner, then the hero banner title is an H1 and is the only H1 on the page. 
4. Given I am accessing a web page, when the hero banner is present, then the hero banner title is styled as PP Neue Montreal font, at 48px font size, and bold styling to match brand guidelines for the hero title. 
5. Given I am accessing a web page with a screen reader, when the screen reader reads out the hero banner, then the reader follows a logical page order and all elements in the banner have appropriate tags
6. Given I am on mobile, when there is a long hero title, then it wraps to a new line as needed. 
7. Given I am on mobile, when the page loads, then the hero H1 is sized at 36px.

**Comment by aaronstewart-dev** [comment]
This ticket is dependent on [1391](https://github.com/aemsites/clarkcountynv/issues/1391 ) development as the hero cannot sit underneath the header until that component is updated.

**Comment by sarah0arnold** [comment]
Story updated with design link and a small tweak to add overlay opacity details
